include/psa/crypto_values.h

```diff
@@ -667,6 +668,40 @@
 /** SHA3-512 */
 #define PSA_ALG_SHA3_512                        ((psa_algorithm_t)0x01000013)
 
+/** Allow any hash algorithm.
+ *
+ * This value may only be used to form the algorithm usage field of a policy
+ * for a signature algorithm that is parametrized by a hash. That is,
+ * suppose that `PSA_xxx_SIGN` is one of the following macros:
+ * - #PSA_ALG_RSA_PKCS1V15_SIGN, #PSA_ALG_RSA_PSS,
+ * - #PSA_ALG_DSA, #PSA_ALG_DETERMINISTIC_DSA,
+ * - #PSA_ALG_ECDSA, #PSA_ALG_DETERMINISTIC_ECDSA.
+ * Then you may create a key as follows:
+ * - Set the key usage field using #PSA_ALG_ANY_HASH, for example:
+ *   ```
+ *   psa_key_policy_set_usage(&policy,
+ *                            PSA_KEY_USAGE_SIGN, //or PSA_KEY_USAGE_VERIFY
+ *                            PSA_xxx_SIGN(PSA_ALG_ANY_HASH));
```

Comment:
`PSA_xxx_SIGN` should be used even when `PSA_KEY_USAGE_VERIFY` is specified, right? `SIGN` is the algorithm type, whether you are signing or verifying. My gut-reading of this was to think we need PSA_xxx_VERIFY, but that's not the case. An alternative like `PSA_xxx_SIGNATURE` might be too verbose, though.

Reply:
I agree that `PSA_xxx_SIGNATURE` is better. Amended.